docs/docs/spec.md

```diff
@@ -609,8 +598,9 @@ rules.
 > Our goal is to allow tools and their users to choose their desired level of
 > strictness in the presence of `UNSPECIFIED`. The basic idea is that, every
 > time a tool encounters a type component with the nullness operator
-> `UNSPECIFIED`, it has the option to fork off two "worlds": one in which the
-> operator is `UNION_NULL` and one in which it is `NO_CHANGE`.
+> `UNSPECIFIED`, it has the option to fork off three "worlds": one in which the
+> operator is `UNION_NULL`, one in which it is `MINUS_NULL`, and one in which it
+> is `NO_CHANGE`.
 >
 > In more detail: When tools lack a nullness specification for a type, they may
 > choose to assume that either of the resulting worlds may be the "correct"
```

Comment:
```suggestion
> choose to assume that any of the resulting worlds may be the "correct"
```

I agree that it should be three options. This reference to "either" should be changed to "any". I didn't look whether there are other references to "two" or "either".